Stage1/environment/env_1d.py

```diff
@@ -20,7 +50,7 @@ def __init__(self, plan_file_path: str):
             0: speed, px/sec
             1: turn, radians 
         """
-        self.action_space = spaces.Box(low=[0, -2 * np.pi], high=[100, 2 * np.pi])
```

Comment:
Please add the comments describing what is going on here, and what these magic numbers are. In general, magic numbers should be changed to named constants or described in the comments.